### DISCUSSION TITLE
Zoom reminders not working

### DISCUSSION THREAD

**Discussion by louisaoc** [isuru89/moodle-local_reminders#157] [Q&A]
I have setup the activity reminders to go out 1 day before for both "local_reminders_dueopenrdays" and "local_reminders_duerdays". I set explicit reminder activation On. I create a zoom activity with Zoom plugin with a meeting date 25 hours from now.
My cron runs every 5min. I expect the activity to be listed on my course specific reminder list, it is not there.
Zoom was not excluded on global settings. Other activities work. Please advise. 
Seems the "When" date in Zoom activity is perhaps not picked up by either open or close date logic?

**Comment by louisaoc** [comment]
![zoom_event_reminder](https://user-images.githubusercontent.com/72609445/191228934-8fd1a278-3d81-449d-8350-2c428f9b0011.png)
It might be just as simple as adding ‘zoom’ to the list of acceptable statuses? But there might be something else down the line, we didn't investigate further.

**Reply by isuru89** [reply]
Anyhow, I would be looking for ways where a user can select applicable event types from the plugin settings page by querying the db and listing it on the page.
But was not successful as I could not find a way to query the DB from the settings page.

So, every time, some third-party module added a new event type to the database, the plugin will not pick those up.

I am also looking into exclusion filtering instead of inclusion, in which we define only what needs to be excluded (Sql `NOT IN`) than included (current `IN`).

**Comment by louisaoc** [comment]
We fixed this bug. Reminders are now working for Zoom. Can we contribute this back?

**Reply by isuru89** [reply]
@louisaoc 
Yes please.

**Reply by louisaoc** [reply]
Our developer, Paul will be in contact. Thank you

**Reply by paulandm** [reply]
Hey @isuru89 , I hope you are well.
I don't know if you've received my emails about the pull request that contains this fix?
I'll send another email just now.
